### ISSUE TITLE
Exited with code 8

### ISSUE THREAD

**Issue by ghost** [huttonr/bootstrap3#4]
I am trying to use the Bootstrap package with Less in latest Meteorjs (1.2.1), so I installed it like so:

`meteor add less`

`meteor added huttonr:bootstrap3`

Then I created `bootstrap-settings.json` and set `customVariables` to true.

If I now run the application I get the following errors:

```
W20151205-15:33:37.312(-3)? (STDERR)          
W20151205-15:33:37.313(-3)? (STDERR) /home/user/.meteor/packages/meteor-tool/.1.1.10.ki0ccv++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:245
W20151205-15:33:37.313(-3)? (STDERR)                        throw(ex);
W20151205-15:33:37.313(-3)? (STDERR)                              ^
W20151205-15:33:37.343(-3)? (STDERR) ReferenceError: jQuery is not defined
W20151205-15:33:37.343(-3)? (STDERR)     at client/lib/bootstrap/bootstrap.generated.js:514:1
W20151205-15:33:37.344(-3)? (STDERR)     at client/lib/bootstrap/bootstrap.generated.js:2346:1
W20151205-15:33:37.344(-3)? (STDERR)     at /home/user/Meteor/seojobportal/.meteor/local/build/programs/server/boot.js:242:10
W20151205-15:33:37.344(-3)? (STDERR)     at Array.forEach (native)
W20151205-15:33:37.344(-3)? (STDERR)     at Function._.each._.forEach (/home/user/.meteor/packages/meteor-tool/.1.1.10.ki0ccv++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/server-lib/node_modules/underscore/underscore.js:79:11)
W20151205-15:33:37.344(-3)? (STDERR)     at /home/user/Meteor/seojobportal/.meteor/local/build/programs/server/boot.js:137:5
=> Exited with code: 8
```

Any idea what is wrong?

**Comment by emgee3** [comment]
The error is that jQuery is undefined on the server. Sounds like your `bootstrap-settings.json` is in the root of your project, thus being processed for both client and server. Try moving your `bootstrap-settings.json` into a folder named `client` in your project. 

@huttonr the readme mentions the file should go in a client folder, but maybe add a check for it anyway?

**Comment by ghost** [comment]
@emgee3 that solved it. Thanks.

**Comment by huttonr** [comment]
https://github.com/huttonr/bootstrap3/issues/5

**Comment by huttonr** [comment]
Fully fixed now with https://github.com/huttonr/bootstrap3/commit/b581b84fa4ea9b17a06acb5676d050a5fba1a449

**Comment by huttonr** [comment]
Now published as part of huttonr:bootstrap3@3.3.5_8

https://github.com/huttonr/bootstrap3/commit/e041ebb40c03343ca2294dcf3eba10a8fe351072
